postgresql/installingpostgresqlonibmi.md

```diff
@@ -0,0 +1,171 @@
+# Install and configure Postgresql DB Server on IBM i 
+
+Install postgres yum packages from IBM i ACS Open Source Package Management
+```
+Package list:
+postgresql12
+postgresql12-contrib
+postgresql12-devel
+postgresql12-docs
+postgresql12-libecpg
+postgresql12-libogtypes
+postgresql12-libpq
+postgresql12-server
```

Comment:
I suspect this is the only one you actually need to install manually; the rest should be pulled in as dependencies automatically.